<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hastaduzenle</class>
 <widget class="QDialog" name="hastaduzenle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hasta Düzenle</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lblAdi">
         <property name="text">
          <string>&amp;Adı</string>
         </property>
         <property name="buddy">
          <cstring>leAdi</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="leAdi"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblSoyadi">
         <property name="text">
          <string>&amp;Soyadı</string>
         </property>
         <property name="buddy">
          <cstring>leSoyadi</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="leSoyadi"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblCinsiyet">
         <property name="text">
          <string>&amp;Cinsiyet</string>
         </property>
         <property name="buddy">
          <cstring>cbCinsiyet</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="cbCinsiyet">
         <item>
          <property name="text">
           <string>ERKEK</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>KADIN</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblDogumTarihi">
         <property name="text">
          <string>&amp;Doğum Tarihi</string>
         </property>
         <property name="buddy">
          <cstring>deDogumTarihi</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDateEdit" name="deDogumTarihi"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblKimlikNo">
         <property name="text">
          <string>&amp;Kimlik No</string>
         </property>
         <property name="buddy">
          <cstring>leKimlikNo</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="leKimlikNo"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="lblTel">
         <property name="text">
          <string>&amp;Telefon No</string>
         </property>
         <property name="buddy">
          <cstring>leTel</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="leTel"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblAdres">
         <property name="text">
          <string>&amp;Adres</string>
         </property>
         <property name="buddy">
          <cstring>teAdres</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QTextEdit" name="teAdres"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnKaydet">
         <property name="text">
          <string>Kaydet</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnVazgec">
         <property name="text">
          <string>Vazgeç</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>422</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0" colspan="3">
      <widget class="QListWidget" name="lwKronikHastaliklar"/>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="cbKanGrubu">
       <item>
        <property name="text">
         <string>A+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>A−</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B−</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AB+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AB−</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0−</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="2">
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnAlerjilerEkle">
         <property name="text">
          <string>Ekle</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnAlerjilerSil">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Sil</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLineEdit" name="leAlerjiler"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblKanGrubu">
       <property name="text">
        <string>&amp;Kan Grubu</string>
       </property>
       <property name="buddy">
        <cstring>cbKanGrubu</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblKronikHastaliklar">
       <property name="text">
        <string>&amp;Kronik Hastalıklar</string>
       </property>
       <property name="buddy">
        <cstring>leKronikHastaliklar</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnKronikHastaliklarEkle">
         <property name="text">
          <string>Ekle</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnKronikHastaliklarSil">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Sil</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QListWidget" name="lwAlerjiler"/>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QLineEdit" name="leKronikHastaliklar"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblAlerjiler">
       <property name="text">
        <string>&amp;Alerjiler</string>
       </property>
       <property name="buddy">
        <cstring>leAlerjiler</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnVazgec</sender>
   <signal>clicked()</signal>
   <receiver>hastaduzenle</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>417</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
